pixelluxe-gui/src/main/java/at/jku/pixelluxe/ui/menu/ContourMenu.java

```diff
@@ -0,0 +1,54 @@
+package at.jku.pixelluxe.ui.menu;
+
+import at.jku.pixelluxe.ui.Component;
+
+import javax.swing.*;
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent;
+
+public class ContourMenu extends JMenu implements Component<JMenu> {
+	private final Runnable onHorizontal;
+	private final Runnable onVertical;
+	private final Runnable onLaplace;
+	private final Runnable onEmboss;
+	private final Runnable onOutline;
+
+    public ContourMenu(Runnable onHorizontal, Runnable onVertical, Runnable onLaplace, Runnable onEmboss, Runnable onOutline) {
+		super("Contour");
+        this.onHorizontal = onHorizontal;
```

Comment:
Strg + Alt + L (Autoformat) in all files pls 🙃

Reply:
Corrected formatting